### PR TITLE
Set up associations

### PR DESCRIPTION
**Pull request by omar25ahmed**
# Learning Objectives:
-  Create classroom class
-  Make student class associated with classroom
- Create the rental class to make the many to many relationships
- Create book class
- Add rental method to the class Person

### CODE REVIEW

File: book.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+require './rental'
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```
Comment (bimbolabuari) [inline book.rb]: Getters and Setters for `@rentals` should be present, Kindly add the rentals to the `attr_accessor`

File: rental.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
```
Comment (bimbolabuari) [inline rental.rb]: Getters and Setters for `@book` and `@person` instance variables should be present, Kindly add them to the `attr_accessor`

File: person.rb [hunk 3]
```diff
@@ -2,14 +2,15 @@
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
```
Comment (bimbolabuari) [inline person.rb]: Getters and Setters for `@rentals` should be present

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @omar25ahmed, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Set up associations between classes and objects
- Correct Gitflow

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @omar25ahmed, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### You did well to make the changes requested 👍

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**